barretenberg/cpp/src/barretenberg/vm/avm/trace/bytecode_trace.cpp

```diff
@@ -3,12 +3,37 @@
 #include "barretenberg/crypto/poseidon2/poseidon2_params.hpp"
 
 namespace bb::avm_trace {
+
+// These separators can be added to the constants file, but it's a bit finicky since there are the GENERATOR_INDEX ones
+// We can migrate these once they are finalised as part of contract address derivation
+const FF PARTIAL_ADDRESS = 27;
+const FF CONTRACT_ADDRESS_V1 = 15;
+const FF CONTRACT_LEAF = 16;
```

Comment:
These constants might disappear/change in the new address derivation scheme

Reply:
Did you already change? will they disappear?